pygotham/frontend/talks.py

```diff
@@ -93,18 +94,29 @@ def proposal(pk=None):
             type='talk',
         )
 
-    form = TalkSubmissionForm(obj=talk)
-    if form.validate_on_submit():
-        form.populate_obj(talk)
+    talk_form = TalkSubmissionForm(obj=talk)
 
+    if talk_form.validate_on_submit():
+        talk_form.populate_obj(talk)
+        speaker = Speaker(
+            user=current_user,
+            talk=talk,
+        )
         db.session.add(talk)
+        db.session.add(speaker)
+
+        for invite_form in talk_form.speaker_invites:
+            if invite_form.validate(invite_form):
+                speaker_invite = SpeakerInvite(talk=talk, **invite_form.data)
+                db.session.add(speaker_invite)
```

Comment:
@jonafato & @dirn I am convinced this is not the right way to do this, but I can't find a better way. `populate_obj` wasn't working. Also line 109 seems strange, if there is another way todo it please let me know.